tool/lib/commands/build_release.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:devtools_tool/model.dart';
+import 'package:io/io.dart';
+import 'package:path/path.dart' as path;
+
+import '../utils.dart';
+
+const _useLocalFlutterFlag = 'use-local-flutter';
+const _updatePerfettoFlag = 'update-perfetto';
+
+class BuildReleaseCommand extends Command {
+  BuildReleaseCommand() {
+    argParser
+      ..addFlag(
+        _useLocalFlutterFlag,
+        negatable: false,
+        defaultsTo: false,
+        help:
+            'Whether to use the Flutter SDK on PATH instead of the Flutter SDK '
+            'contained in the "tool/flutter-sdk" directory.',
+      )
+      ..addFlag(
+        _updatePerfettoFlag,
+        negatable: false,
+        defaultsTo: false,
+        help: 'Whether to update the Perfetto assets before building DevTools.',
+      );
+  }
+
+  @override
+  String get name => 'build-release';
+
+  @override
+  String get description => 'Prepares a release build of DevTools.';
+
+  @override
+  Future run() async {
+    final repo = DevToolsRepo.getInstance();
+    final processManager = ProcessManager();
+
+    final useLocalFlutter = argResults![_useLocalFlutterFlag];
+    final updatePerfetto = argResults![_updatePerfettoFlag];
+
+    if (!useLocalFlutter) {
+      logStatus('updating tool/flutter-sdk to the latest flutter candidate');
+      await processManager.runProcess(CliCommand.tool('update-flutter-sdk'));
+    }
+
+    if (updatePerfetto) {
+      logStatus('updating the bundled Perfetto assets');
```

Comment:
suggestion: should the comment for now to be "skipping updating the bundled Perfetto assets" so we aren't confused why the perfetto assets don't get updated?